filament/src/IndirectLight.cpp

```diff
@@ -67,6 +67,26 @@ IndirectLight::Builder& IndirectLight::Builder::irradiance(uint8_t bands, float3
     return *this;
 }
 
+IndirectLight::Builder& IndirectLight::Builder::radiance(uint8_t bands, float3 const* sh) noexcept {
```

Comment:
Doesn't this mean deeplight cannot call this method directly since they already have the coefficients below baked in their values?

Reply:
I'm not following. The coefficients this methods applies are just the convolution by <n.l>, the pre-scaling by the normalization factors and the diffuse brdf 1/pi. None of those are pre-baked.